frontend/packages/kubevirt-plugin/src/components/vms/vmi-details-page.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { navFactory, HintBlock, ExternalLink } from '@console/internal/components/utils';
+import { DetailsPage } from '@console/internal/components/factory';
+import { PodModel } from '@console/internal/models';
+import { VMDisksFirehose } from '../vm-disks';
+import { VMNics } from '../vm-nics';
+import {
+  VirtualMachineInstanceMigrationModel,
+  VirtualMachineInstanceModel,
+  VirtualMachineModel,
+} from '../../models';
+import { getResource } from '../../utils';
+import {
+  VM_DETAIL_OVERVIEW_HREF,
+  VM_DETAIL_DISKS_HREF,
+  VM_DETAIL_NETWORKS_HREF,
+  VM_DETAIL_CONSOLES_HREF,
+} from '../../constants';
+import { VMEvents } from './vm-events';
+import { VMConsoleFirehose } from './vm-console';
+import { VMDetailsFirehose } from './vm-details';
+import { menuActionsCreator } from './menu-actions';
+import { VMDashboard } from './vm-dashboard';
+
+import './vmi-details-page.scss';
+
+export const VirtualMachinesInstanseDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (
+  props,
+) => {
+  const { name, ns: namespace } = props.match.params;
+
+  const breadcrumbsForVMPage = (match: any) => () => [
+    {
+      name: VirtualMachineModel.labelPlural,
```

Comment:
yes, the idea is to have the same page ... for vm and vmi, It do have different path now, but user should see the same breadcrumbs for both ... 

p.s.
when changing namespace it will get to the wrong list, but this will have to be looked at at later time.

Reply:
for the detail, shouldn't hurt removing the namespace select altogether

Reply:
makes sense to me too, this is a console wise change, so not for this PR, I agree we should look how to do this.

Reply:
ok